Sources/SPMUtility/ArgumentParser.swift

```diff
@@ -40,8 +40,11 @@ extension ArgumentParserError: LocalizedError {
 extension ArgumentParserError: CustomStringConvertible {
     public var description: String {
         switch self {
-        case .unknownOption(let option):
-            return "unknown option \(option); use --help to list available options"
+        case .unknownOption(let option, let suggestion):
+            return [
+                "unknown option \(option); use --help to list available options",
+                suggestion.map { "Did you mean \($0)?" }
+            ].compactMap({ $0 }).joined(separator: "\n")
```

Comment:
How about we simplify this to:

```swift
var desc = "unknown option \(option); use --help to list available options"
suggestion.map { desc + "\nDid you mean \($0)?" }
return desc
```

Reply:
Or a simple `if let suggestion` sounds good too